GliaWidgets/SecureConversations/SecureConversations.WelcomeStyle.swift

```diff
@@ -3,40 +3,125 @@ import UIKit
 
 extension SecureConversations {
     /// Theme of the secure conversations welcome view.
-    public class WelcomeStyle {
+    public struct WelcomeStyle: Equatable {
         /// Style of the view's header (navigation bar area).
         public var header: HeaderStyle
 
         /// Title shown in the header.
-        public var title: String
+        public var headerTitle: String
 
         /// Title shown in the welcome area.
-        public var welcomeTitle: String
+        public var welcomeTitleStyle: TitleStyle
 
         /// Description showm in the welcome area.
-        public var welcomeDescription: String
+        public var welcomeSubtitleStyle: SubtitleStyle
+
+        public var checkMessagesButtonStyle: CheckMessagesButtonStyle
+
+        public var messageTitleStyle: MessageTitleStyle
+
+        public var messageTextViewStyle: MessageTextViewStyle
+
+        public var sendButtonStyle: SendButtonStyle
 
         /// View's background color.
         public var backgroundColor: UIColor
 
         /// - Parameters:
         ///   - header: Style of the view's header (navigation bar area).
-        ///   - title: Title shown in the header.
+        ///   - headerTitle: Title shown in the header.
```

Comment:
Also here below we can add the additional parameters that are needed.

Reply:
Addressed in d13ff2b076b1fb2dd2347ac2679f10961f2a0b7b.